TrkDiag/src/TrackQuality_module.cc

```diff
@@ -16,8 +16,6 @@
 #include "ProditionsService/inc/ProditionsHandle.hh"
```

Comment:
I think we should consider whether TrackQuality and TrackPID belong in Offline, or should be moved to the TrkAna repo.  It's not a critical issue for this PR.

Reply:
There are two important use cases to consider.

1.   How do we define good tracks for the purpose of running validation?  My vision of the future is that there may be several (many?) good track algorithms and most will live in analysis repos; and I think that final analysis will use one of these. 
2.  During production there will be some minimal quality cut to decide what tracks cross the threshold to pass the event. 

It's not clear to me if one algorithm can do both jobs; maybe one algorithm with two different cut sets?

In both cases I see two options: one, we retain some notion of good quality inside Offline with the understand that it's supposed to be good enough for the two use cases above;  two, running validation or running production requires that we load another repo just to get track quality.  My preference is the first.